ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -42,6 +45,12 @@ Button {
     QtObject {
         id: d
         readonly property color textColor: root.enabled || root.loading ? root.textColor : root.disabledTextColor
+        readonly property Component statusIcon: StatusIcon {
```

Comment:
So.. why don't you put a toplevel `alias` for the `iconLoader`, and initialize it inline with this `statusIcon` component?

Reply:
Right, that I'm also unsure about; I just proposed an alternative solution, wdyt @alexandraB99

Reply:
@IvanBelyakoff totally understandable! I'd move on with the boolean as there's no need to expose the whole component here, but it's just about how the icon will be rendered. would you be fine changing it?